cfy_manager/components/restservice/db.py

```diff
@@ -222,32 +222,22 @@ def _create_process_env(rest_config=None, authorization_config=None,
     return env
 
 
-def _run_script(script_name, args_dict=None, configs=None):
+def _run_script(script_name, script_input=None, configs=None):
     """Runs a script in a separate process.
 
     :param script_name: script name inside the SCRIPTS_PATH dir.
-    :param args_dict: script arguments to pass to the script.
+    :param script_input: script input to pass to the script.
     :param configs: keword arguments dict to pass to _create_process_env(..).
     :return: the script's returned when it finished its execution.
     """
-    env_dict = None
-    if configs is not None:
-        env_dict = _create_process_env(**configs)
+    configs = configs or {}
+    env_dict = _create_process_env(**configs)
```

Comment:
it could not stay None?

Reply:
Changed to `_create_process_env(**configs) if configs else None`